sdk/healthdataaiservices/Azure.ResourceManager.HealthDataAIServices/samples/Generated/Samples/Sample_DeidServiceCollection.cs

```diff
@@ -23,7 +23,7 @@ public partial class Sample_DeidServiceCollection
         public async Task CreateOrUpdate_DeidServicesCreateGeneratedByMaximumSetRuleStable()
         {
             // Generated from example definition: 2024-09-20/DeidServices_Create_MaximumSet_Gen.json
-            // this example is just showing the usage of "DeidService_Create" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "DeidServices_create" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `create`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/healthdataaiservices/HealthDataAIServices.Management/main.tsp#L125